docs/shared/connectors/metaMask.md

```diff
@@ -10,11 +10,30 @@ Connector for the [MetaMask SDK](https://github.com/MetaMask/metamask-sdk).
 
 ## Import
 
-::: warning WARNING This connector has a large file size due to the underlying [`@metamask/sdk`](https://github.com/MetaMask/metamask-sdk). 
-For mobile support, it is recommended to use <a :href="`/${docsPath}/api/connectors/walletConnect`">`walletConnect`</a>. For desktop support, you should rely on <a :href="`/${docsPath}/api/createConfig#multiinjectedproviderdiscovery`">Multi Injected Provider Discovery</a> (EIP-6963) via the Wagmi <a :href="`/${docsPath}/api/createConfig#config`">`Config`</a>.
-
-On desktop, if you must target MetaMask, you can use <a :href="`/${docsPath}/api/connectors/injected`">`injected`</a>, e.g. <a :href="`/${docsPath}/guides/migrate-from-v1-to-v2#removed-metamaskconnector`">`injected({ target: 'metaMask' })`</a>. Please keep in mind this approach does not use EIP-6963 and other injected wallets could conflict with the connector.
-:::
+>[!WARNING]  
+> This connector has a large file size due to the underlying @metamask/sdk.
+
+>[!WARNING]  
+> MetaMask SDK can be used for mobile and desktop support.
+
+>[!WARNING]  
+> MetaMask SDK supports EIP-6963, which prevents other injected wallets to conflict with the connector.
+
+>[!WARNING]
+> Due to a limitation in iOS, Safari cancels deep link connections if they do not happen within 500ms. It's recommended to use a direct RPC method for transactions to avoid this issue. Instead of using `getSigner` and `sendTransaction`, directly invoke the `send` method with `'eth_sendTransaction'` and wait for the transaction using `waitForTransaction`. This approach avoids the deep link timeout issue on iOS Safari.
```

Comment:
Does MetaMask SDK use Universal Links or Deep Links? 

If using Deep Links, this shouldn’t happen.

Reply:
Some other context: WalletConnect had previously favored Universal Links where this behavior would occur (which we mitigated with prepare hook), however after the Deep Link migration, this didn’t happen anymore (a prompt to open the app shows instead).

Reply:
Check out here for more info: https://github.com/WalletConnect/walletconnect-docs/blob/6307aa2266ff008b2af4a5fe25406a955544dd47/docs/web3modal/advanced/mobile-linking.md#universal-and-deeplinking